tiledb/sm/filesystem/vfs.cc

```diff
@@ -651,6 +731,10 @@ Status VFS::max_parallel_ops(const URI& uri, uint64_t* ops) const {
     RETURN_NOT_OK(
         config_.get<uint64_t>("vfs.azure.max_parallel_ops", ops, &found));
     assert(found);
+  } else if (uri.is_gcs()) {
+    RETURN_NOT_OK(
+        config_.get<uint64_t>("vfs.gcs.max_parallel_ops", ops, &found));
```

Comment:
Perhaps we should unify this parameter before 1.8 release? I'm not sure there is a case where users would want to set it separately for different VFS, and simplifying the config/config docs is good whenever possible.

Reply:
I'm in favor of that but the counter-argument would be that max parallel ops only applies to cloud-backends, so our config naming options are:
1. `vfs.max_parallel_ops` -- misleading scope because it doesn't universally apply to all VFS backends
2. `vfs.cloud.max_parallel_ops` -- misleading because the `cloud` scope isn't consistent with our other config options where the second scope should name a VFS backend
3. as-is

If you prefer option 1 or 2, let me know and I'll create a ticket to investigate it later

Reply:
Hmm, on second thought let’s just leave it (IMO) — as long as the default is `vfs.num_threads` like the other ones, then people have a knob to change this already at a higher-level. Fine-tuning it more specifically will probably be a rare/advanced option.